pkg/apis/addtoscheme_cc_ccc_v1beta.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package apis
+
+import (
+	opcorev1beta1 "github.com/GoogleCloudPlatform/k8s-config-connector/operator/pkg/apis/core/v1beta1"
+)
+
+func init() {
+	// register CC, CCC kinds for resourceactuation.ShouldActuate
+	AddToSchemes = append(AddToSchemes, opcorev1beta1.SchemeBuilder.AddToScheme)
```

Comment:
This is not the normal way, and I worry that it's non-deterministic.  Are we doing this elsewhere?

Reply:
I think this is the current pattern for the `iam` schemes in the same folder;

however, with https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/1280/files, I think I will undo this change and just rebase onto that since it already solves for adding the CC/ CCCs to the mgr's scheme in kccmanager